### PR TITLE
Update: Added Dropdown labeling (refs #147)

### PR DESCRIPTION
**Pull request by hanshillen**
### Update

* Labeled the Matching dropdown buttons with the match item text using `aria-labelledby`

### CODE REVIEW

**Comment by hanshillen** [comment]
Alternatively you could use `aria-label` with the actual question text , then let the author override this with an `_ariaLabel` field for each option if needed.

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

File: templates/matching.jsx [hunk 1]
```diff
@@ -43,6 +43,7 @@ export default function Matching(props) {
         }, index) => {
           const activeOption = _options.find(option => (option._itemIndex === _index) && option._isActive);
           const displayItemAsCorrect = (!_isEnabled && _shouldShowMarking && (_isCorrectAnswerShown || activeOption?._shouldBeSelected));
+          const _questionTitleId = `${_id}-matching-item-${_index}__title`;
```
Comment (oliverfoster) [inline templates/matching.jsx:46]:
```suggestion
          const questionTitleId = `${_id}-matching-item-${_index}__title`;
```

Could you remove the `_` throughout please? It's only really loosely used to identify non-translatable properties in the JSON.

Comment (hanshillen) [reply templates/matching.jsx]: Done

**Review by kirsty-hames — COMMENTED** [review]
Thanks @hanshillen. This aligns with the WCAG example using the question to label the listbox. Note, `aria-labelledby` takes precedence over all other methods of providing an accessible name so the inner text is no longer read e.g. 'Select an option' or the selected option. 
You can combine more than one element id into a single accessible name however so I'd suggest adding the inner text to the label too. 

For example:
`aria-labelledby={props.questionTitleId + ' ' + '${_id}-matching-item-${_itemIndex}__text'}`

Add id `${_id}-matching-item-${_itemIndex}__text`} to `.dropdown__inner`

When the dropdown btn receives focus, this should read:
_question text_ + _'Select an option' / selected option_ + _listbox popup_

**Review by oliverfoster — APPROVED** [review]

**Pull request merged**